homeassistant/components/mystrom/sensor.py

```diff
@@ -44,13 +50,40 @@ class MyStromSwitchSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfPower.WATT,
         value_fn=lambda device: device.consumption,
     ),
+    MyStromSwitchSensorEntityDescription(
+        key="energy_since_boot",
+        translation_key="energy_since_boot",
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        value_fn=lambda device: None
+        if device.energy_since_boot is None
+        else device.energy_since_boot / 3600000,
+    ),
     MyStromSwitchSensorEntityDescription(
         key="temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         value_fn=lambda device: device.temperature,
     ),
+    MyStromSwitchSensorEntityDescription(
+        key="time_since_boot",
+        translation_key="time_since_boot",
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        value_fn=lambda device: device.time_since_boot,
+        suggested_display_precision=0,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
Instead, we should calculate the datetime of when this boot happened and show it as a timestamp sensor, as that would only write state once per boot and not once per update

Reply:
Added `boot_id` as an additional state attribute to the `time_since_boot` (Last Restart) sensor. The value is only updated when a new boot (ID) occurs. Please review again. Thank you!